paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -799,6 +851,7 @@ def on_enter_deploying(self):
                 target=self.do_wait_for_deployment, args=(self.commit,), daemon=True
             )
             thread.start()
+            self.schedule_paasta_status_reminder()
 
     def on_exit_deploying(self):
         self.wait_for_deployment_green_lights[self.commit].clear()
```

Comment:
you could throw some code to cancel the reminder here

Reply:
In order to do that though, I would have to build up the same kind of `timer_handle` logic that stitch does, right?

Reply:
yeah, youd have to save the return value of call_later.

Reply:
ok. I'm just in general hesitant to turn this functionality into its own full state machine

Reply:
Ok. Cancelation support added.